main.py

```diff
@@ -2,6 +2,48 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    actors = [
+        ("George", "Clooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    genres = ["Western", "Action", "Drama"]
+
+    for item_first, item_second in actors:
+        Actor.objects.create(
+            first_name=item_first,
+            last_name=item_second)
+
+    for item in genres:
+        Genre.objects.create(name=item)
+
+    Genre.objects.filter(name="Dramma",).update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George ",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action",
+    ).delete()
```

Comment:
please use same style in all code:

in this case you make a block

    Genre.objects.filter(
        name="Action",
    ).delete()

but in same situation you make one line
```
Genre.objects.filter(name="Dramma",).update(name="Drama")
```